assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -524,13 +524,15 @@ che.workspace.plugin_registry_url=https://che-plugin-registry.prod-preview.opens
 # In case Che plugins tooling is not needed value 'NULL' should be used
 che.workspace.devfile_registry_url=https://che-devfile-registry.prod-preview.openshift.io/
 
-# Defines a default value for persist volumes that clients like Dashboard
-# should propose for users during workspace creation.
-# Possible values: true or false
-# In case of true - PersistentVolumeClaims are used by declared volumes by user and plugins. `true`
-# value is supposed not to be set explicitly in Devfile attributes since it's default fixed behaviour.
-# In case of false - emptyDir is used instead of PVCs. Note that data will be lost after workspace restart.
-che.workspace.persist_volumes.default=true
+# The configuration property that defines available values for storage types that clients like
+# Dashboard should propose for users during workspace creation/update.
+# In case client not support is not support this value 'NULL' should be used
+che.workspace.storage.available_types=persistent,ephemeral,async
```

Comment:
Can we remove async from here and put 7.16 as a milestone?

Reply:
Please add a short description for types we have, for admin easier decide which they would like to enable/disable